AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/test_AddSameAssetsDifferentNames_ShouldProcess/test_prefab - Copy.prefab

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this a copy that was accidentally submitted?

Reply:
Each test looks at a different test asset folder, I could update these to all point to a single file later on and limit the number of copies in the test asset folders. 

As for the "WWWWW" prefabs, it's to cause the test to generate an error, then correct the error by fixing the prefab by re-writing it into a working JSON file - Originally these used a corrupted slice to achieve the same results.